src/config.js

```diff
@@ -67,7 +67,7 @@ export default class Config {
 	 *		config.get( 'toolbar.color' ); // 'red'
 	 *
 	 * @param {String|Object} name The configuration name or an object from which take properties as
-	 * configuration entries. Configuration names are case-insensitive.
+	 * configuration entries. Configuration names are case-sensitive.
```

Comment:
I'm not sure if information about case-sensitive is needed, because JavaScript is case-sensitive, but let us wait for @Reinmar.